examples/macro/g3g4libs.C

```diff
@@ -0,0 +1,185 @@
+//------------------------------------------------
```

Comment:
This macro should be removed, and the existing macros g3libs.C and g4libs.C should be adapted so that they can be both included in load_multi.C. The best seems to me just to add the definition of isLibrary function, which is present in both macros and causes a clash, into a namespace different in each g3/g4libs.C